dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/SchemaDotOrgExporter.java

```diff
@@ -74,9 +74,13 @@ public SchemaDotOrgExporter(String dataverseSiteUrlStatic) {
 
     // -------------------- LOGIC --------------------
 
+    public String exportDataset(DatasetVersion version, String HideSchemaDotOrgDownloadUrls) {
+        return version.getJsonLd(dataverseSiteUrlStatic, HideSchemaDotOrgDownloadUrls);
+    }
+
     @Override
-    public String exportDataset(DatasetVersion version) {
-        return version.getJsonLd(dataverseSiteUrlStatic);
+    public String exportDataset(DatasetVersion version) throws ExportException {
+        return null;
```

Comment:
1. You should never return null.
2. This will never work, the point of the strategy pattern is not to be concerned with implementation and to use interface, and if you will use this interface, you won't be able to export anything here.